Braintree.xcodeproj/project.pbxproj

```diff
@@ -1413,6 +1415,7 @@
 				BEDB820929B7A9E600075AF3 /* BTVenmoClient.swift */,
 				BE0AF6BA29AFD53300245C2C /* BTVenmoError.swift */,
 				BE0AF6BE29AFF13200245C2C /* BTVenmoRequest.swift */,
+				096C6B2529CCDCEB00912863 /* BTVenmoLineItem.swift */,
```

Comment:
We typically sort our files by name. If you right click the directory in Xcode you should see the option "sort by name" towards the bottom

Reply:
Hmm I can't seem to find this file `project.pbxproj` in my local repository... 🤔